### DISCUSSION TITLE
OutAllPlanes in FAST.Farm not working

### DISCUSSION THREAD

**Discussion by MYMahfouz** [OpenFAST/openfast#1722] [General]
Dear Jason,

I have set `OutAllPlanes = True`, in FAST.Farm with the polar model and the wake planes were not printed.

The folder `vtk_ff_planes` was created but it remained empty.

Best regards,

**Comment by jjonkman** [comment]
@ebranlard -- Are you aware of an issue associated with `OutAllPlanes` in FAST.Farm?

**Reply by ebranlard** [reply]
I'm not aware of any issue, it used to work..

**Comment by ebranlard** [comment]
I believe there are two issues that can potentially affect the outputs for OutAllPlanes:

- Using OpenMP. We have added a OpenMP for loop around the WD_CalcOutput, which is problematic for the GetNewUnit function, and result in the same units being used at the same time.
- A small issue in how we handle the variable p%OutFileRoot. This variable should be replaced by the basename for the output to work correctly. It shouldn't be an issue if FAST.Farm is called from the command line though with a relative path to the input file.

You can try running with a FAST.Farm executable that does not have OpenMP. And make sure you call the exe as follow: `FAST.Farm.exe  inputfile.fstf `. We'll try to address the two issues in a future release.
